src/Simplic.FileStructure/Model/Directory.cs

```diff
@@ -88,5 +88,10 @@ public override int GetHashCode()
         {
             return Id.GetHashCode();
         }
+
+        /// <summary>
+        /// Gets or sets the workflow id 

```

Comment:
Gets or sets the workflow id. The id will be used to identify the workflow configuration for specific directory.